canvas_account_admin_tools/settings/base.py

```diff
@@ -263,9 +263,17 @@
 ICOMMONS_REST_API_TOKEN = SECURE_SETTINGS.get('icommons_rest_api_token')
 ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = False
 
-PERMISSION_ACCOUNT_ADMIN_TOOLS = 'account_admin_tools'
-PERMISSION_PEOPLE_TOOL = 'people_tool'
-PERMISSION_XLIST_TOOL = 'cross_listing'
+# TOOL_PERMISSIONS is used to map app names to SchoolPermission.permission names
+# to determine visibility of tools on dashboard and access to tool views/pages
+TOOL_PERMISSIONS = {
+    # NOTE: canvas_account_admin_tools permission name is shortened to
+    # account_admin_tools due to SchoolPermission.permission field max length
```

Comment:
We could leave this dict as-is, and allow long key names to translate down to shorter ones for the sake of readable permission names in the SchoolPermission tables. The set of SchoolPermissions that need to be displayed to the user in TLT-2651 would be the `values()` of this dict. We could also simplify this to a `list` that just contains the values of this dict here, and have the `views` code use the same name that's in the list. Then the simplified `TOOL_PERMISSIONS` list can just be the keys. We lose the ability to make the dict keys be more meaningful, but a simple code comment can clarify a funky/terse/abbreviated key name, or we could even change the `SchoolPermission` model to accept permissions with names longer than 25 characters (not sure where that came from in the first place).

I lean towards a list here, with code comments to clarify any confusing permissions. Then referring to them from `lti_permissions` doesn't require the `values()` call (see e.g. https://github.com/Harvard-University-iCommons/django-icommons-common/pull/157/files#diff-1769bdf11052a9b2c8f7db98d3e9bdb4R12).